### ISSUE TITLE
crashes on iPhone 7 plus iOS 10.1.1

### ISSUE THREAD

**Issue by jkole-4** [Enthri/iBox-2#2]
App will load up but always crashes after windows xp loading screen. I did edit the min requriement to install on iOS 10.1.1. Is it goner support all of iOS 10 or only iOS 10.2 and up.

**Comment by KyujuuAlpha** [comment]
Have you tried using a different display adapters? The driver may be the issue.

**Comment by Directory** [comment]
How did you make it run on 10.1.1. My 7 is jailbroken and i need this feature so badly. Please help me!

**Comment by ghost** [comment]
Yeah,I can't find it in cydia,or ipa or deb file.and can this emulator emulate an android system on iOS?,seems very awesome.

**Comment by ghost** [comment]
I had the same issue. I fixed it by setting display to None, instead of VBE.

**Comment by eilpx** [comment]
I managed to compile the ipa when Swift 2.3 was available... Tell me if you want the ipa.

**Comment by XJ-Zhang-Th** [comment]
> I managed to compile the ipa when Swift 2.3 was available... Tell me if you want the ipa.

Are you still there, can you provide the IPA file?
